src/share/grabbag/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+add_library(grabbag STATIC
+    alloc.c
+    cuesheet.c
+    file.c
+    picture.c
+    replaygain.c
+    seektable.c
+    snprintf.c)
+target_link_libraries(grabbag
+    FLAC-static
+    replaygain_analysis)
```

Comment:
```suggestion
    replaygain_analysis
    utf8)
```

Reply:
I've checked grabbag, it depends on win_utf8_io rather than utf8